user-guide/Advanced_Modules/SRM/Functions_XML_files.md

```diff
@@ -64,6 +65,11 @@ The file includes the following tags:
 
 - **Version**: Contains the version number of the functions file.
 
+- **OverrideTimeoutVF**: Available from DataMiner 10.5.3/10.6.0 onwards<!--RN 41388-->. When set to "true", this tag allows the timeout override for a Virtual Function (VF).
+
+  > [!NOTE]
+  > Prior to DataMiner 10.5.3/10.6.0, when the [`overrideTimeoutDVE` option](xref:Protocol.Type-overrideTimeoutDVE) is enabled in a *protocol.xml* file, the timeout will apply to DVE elements as well as Virtual Functions. From DataMiner 10.5.3/10.6.0 onwards<!--RN 41388-->, this option will only apply to DVE elements, and the *OverrideTimeoutVF* tag in a *Functions.xml* file can be used to override the timeout for VFs.
+
 - **Function**: Defines a function for the protocol. This tag can have the following attributes:
```

Comment:
Side note: It would be more consistent that the documentation about the functions xml file would be documented as an entry in https://docs.dataminer.services/develop/schemadoc/index.html

Reply:
This would indeed be nice; however, it's not something we can give much priority right now, so it's not likely that this will get done any time soon.